cmd/server.go

```diff
@@ -280,6 +281,10 @@ var boolFlags = map[string]boolFlag{
 		description:  "Disable atlantis auto planning feature",
 		defaultValue: false,
 	},
+	EnablePolicyChecksFlag: {
+		description:  "Enable atlantis to run user defined policy checks. If TFE/TFC is used this is disabled even if set to true. This is due to the fact that TFE does not allow to get plan files.",
```

Comment:
```suggestion
		description:  "Enable atlantis to run user defined policy checks.  This is explicitly disabled for TFE/TFC backends since plan files are inaccessible. ",
```